freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/KerberosMgmt/doc/KeytabModelNotes.java

```diff
@@ -0,0 +1,14 @@
+package com.sequenceiq.freeipa.api.v1.KerberosMgmt.doc;
+
+public class KeytabModelNotes {
+    public static final String CREATE_HOST_AND_PRINCIPAL_NOTES  = "Creates the host if it doesn't exist and also creates the service principal for the given " +
+            "service";
```

Comment:
It will also create the keytab and retrieve it. You may also want to note that if this is invoked multiple times, all prior keytabs are invalidated. (See my prior comment about the `ipa-getkeytab` man page warning.

Reply:
done.